edge_computing/installing_with_siteconfig_operator/cnf-understanding-siteconfig-operator.adoc

```diff
@@ -0,0 +1,37 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cnf-understanding-siteconfig-operator"]
+= Understanding the {sco}
+include::_attributes/common-attributes.adoc[]
+:context: siteconfig-operator
+
+toc::[]
+
+With the new {sco}, you can deploy clusters with all available installation methods.
+The {sco} uses the unified `ClusterInstance` API, which decouples parameters that define a cluster from the manner in which the cluster is deployed.
+This separation removes certain limitations that are presented by the `SiteConfig` kustomize plugin (`siteconfig-v1`) in the current xref:../../edge_computing/ztp-deploying-far-edge-sites.adoc#ztp-deploying-far-edge-sites[{ztp-first} flow], such as agent cluster installations and scalability constraints posed by ArgoCD.
+Thanks to the unified `ClusterInstance` API, the {sco} offers the following improvements:
+
+Isolation:: Separates cluster definition from the installation method. The {sco} CRD, the `ClusterInstance` CR, captures the cluster definition, while the cluster templates capture the cluster architecture and installation methods.
```

Comment:
Is there some repetition here? Should it just be, `The `ClusterInstance` CR,`

Reply:
It's the first time mentioning cluster templates, but the use of the definite article in "...the cluster templates..." makes it sound to me like they were discussed previously. Perhaps you could introduce the concept in the intro. Or change this sentence slightly: "A set of cluster templates capture...."